CHANGELOG.md

```diff
@@ -883,7 +883,42 @@ Return argument value as field definition runtimeType if requested as object typ
 - Diagnostic source which can be used to track field execution times and other events.
 - Implementing a directive middleware has now become much easier with this release. We have built the authorize-directive with these new APIs.
 
-[unreleased]: https://github.com/ChilliCream/graphql-platform/compare/0.8.2...HEAD
+[unreleased]: https://github.com/ChilliCream/graphql-platform/compare/12.0.1...HEAD
+[12.0.1]: https://github.com/ChilliCream/graphql-platform/compare/12.0.0...12.0.1
+[12.0.0]: https://github.com/ChilliCream/graphql-platform/compare/11.1.0...12.0.0
+[11.1.0]: https://github.com/ChilliCream/graphql-platform/compare/10.0.9...11.1.0
```

Comment:
Are these links correct? comparing v10 with v11?

Reply:
Some of them did seem odd to me too, but this is the pattern used by the original CHANGELOG prior to 9.0.0, comparing the current release with the one immediately preceding it:

```markdown
[0.8.2]: https://github.com/ChilliCream/graphql-platform/compare/0.8.1...0.8.2
[0.8.1]: https://github.com/ChilliCream/graphql-platform/compare/0.8.0...0.8.1
[0.8.0]: https://github.com/ChilliCream/graphql-platform/compare/0.7.0...0.8.0
[0.7.0]: https://github.com/ChilliCream/graphql-platform/compare/0.6.11...0.7.0
[0.6.11]: https://github.com/ChilliCream/graphql-platform/compare/0.6.10...0.6.11
```

I am aware there is a host of other releases in between that are simply not listed in this document, but I didn't want to go into back-porting them not knowing if this document even has a future. :)

Reply:
That said, you are right, this is indeed a typo. Fixing it now.

Reply:
Should be good now.